tools/cold-backup/README.md

```diff
@@ -22,36 +25,35 @@ Consumer group offsets are included, but not Kafka Connect, MirrorMaker and Kafk
 - enough disk space
 
 ## Test procedure
+
 ```sh
+CLIENT_IMAGE="quay.io/strimzi/kafka:latest-kafka-2.8.0"
```

Comment:
Should this used the `latest` image? Wouldn't it be better to use some released image which will not change?

Reply:
Yep.